### ISSUE TITLE
Support for CLI visualiser

### ISSUE THREAD

**Issue by wilfredjonathanjames** [clangen/musikcube#174]
Hi there,

This is more of a feature suggestion, but are you planning on supporting / do you already support a CLI visualiser mode? Even an equaliser along the bottom of the interface would be awesome.

Yours,
A terminal head

P.s. truly enjoying `musikcube`. Wonderful project.

**Comment by wijonja** [comment]
In fact, perhaps adding support for https://github.com/dpayne/cli-visualizer would be enough.

**Comment by clangen** [comment]
Yeah, I'm trying to figure out some sort of cool/simple/elegant visualizer that I can build in. It's something I have planned for the future. 

That said, I've never heard of cli-visualizer, but I just tried it -- it seems to work out of the box as long as you're using PulseAudio! Very cool project. I'm going to spend some time looking at the sources to see if I can get it directly integrated easily.

**Comment by wijonja** [comment]
That's awesome, thank you. I couldn't get it working with pulseaudio on my
end when I tried but I'll give it another shot.

On Jan 8, 2018 08:56, "casey langen" <notifications@github.com> wrote:

Yeah, I'm trying to figure out some sort of cool/simple/elegant visualizer
that I can build in. It's something I have planned for the future.

That said, I've never heard of cli-visualizer, but I just tried it -- it
seems to work out of the box as long as you're using PulseAudio! Very cool
project. I'm going to spend some time looking at the sources to see if I
can get it directly integrated easily.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/clangen/musikcube/issues/174#issuecomment-355847906>,
or mute the thread
